gnomad/utils/transcript_annotation.py

```diff
@@ -205,3 +214,199 @@ def tissue_expression_ht_to_array(
         )
 
     return ht
+
+
+def preprocess_variants_for_tx(
```

Comment:
I'm not sure I like this name, maybe `tx_filter_csqs` or `tx_filter_variant_csqs` or `tx_filter_variants_by_csq` or something else like that